cogs/welcome.py

```diff
@@ -38,32 +37,35 @@ async def on_voice_state_update(self, member, before, after):
                         last_seen=now
                         )
                     session.add(user)
+                    userList.append(user)
                 elif (now - user.last_seen).total_seconds() < 3600 * 12:
                     continue
-            vc = await after.channel.connect()
-
-            day_period = get_day_period()
-
-            f = None
-            if day_period == DayPeriod.MORNING:
-                 f = 'data/welcome/mourao.mp3'
-            elif now.hour >= 12 and now.hour < 13:
-                 f = 'data/welcome/dilma.mp3'
-            elif day_period == DayPeriod.AFTERNOON:
-                f = 'data/welcome/jornalhoje.mp3'
-            else:
-                f = 'data/welcome/bonner.mp3'
-
-            audio = FFmpegPCMAudio(f)
-
-            vc.play(audio)
-
-            while vc.is_playing():
-                await asyncio.sleep(.1)
-            await vc.disconnect()
-            for user in userList:
-                user.last_seen = arrow.now('America/Sao_Paulo')
-            session.commit()
+                elif not onlineMember.voice.self_deaf:
```

Comment:
Maybe we could separate this and the if on line 41 from the if that checks if the user exists. This could make the code a bit easier to understand, something like:
```python
                if not user:
                    user = User(
                        member_id=onlineMember.id,
                        last_seen=now
                        )
                    session.add(user)
                    userList.append(user)
                    continue
                    
                if (now - user.last_seen).total_seconds() < 3600 * 12:
                    continue
                elif not onlineMember.voice.self_deaf:
                    userList.append(user)
```

Reply:
I agree with my fellow developer Nixsm

Reply:
Yep, very nice input so we can make our code more readable overall!

Reply:
tchuu tchuuu 🚂